src/System.Net.HttpListener/tests/HttpListenerContextTests.cs

```diff
@@ -43,7 +51,7 @@ public static IEnumerable<object[]> SubProtocol_TestData()
             yield return new object[] { new string[] { "MyProtocol1", "MyProtocol2" }, "MyProtocol2" };
         }
 
-        [ActiveIssue(20246)] // CI hanging frequently
+        [ActiveIssue(20246, TestPlatforms.AnyUnix)] // CI hanging frequently
```

Comment:
I don't understand when you're choosing to use AnyUnix vs IsWindowsImplementation.  I disabled this everywhere because it repros regardless of platform when using the managed implementation.